Tools/ardupilotwaf/boards.py

```diff
@@ -57,6 +57,11 @@ def configure_env(self, cfg, env):
         # make easy to override them. Convert back to list before consumption.
         env.DEFINES = {}
 
+        env.prepend_value('INCLUDES', [
+            cfg.srcnode.find_dir('libraries/AP_Common/missing').abspath(),
+            cfg.srcnode.find_dir('build').abspath()
```

Comment:
you want the bldnode, not hardcoded to 'build'... also because we default to 'build/<board_name>'